<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutwindow</class>
 <widget class="QWidget" name="aboutwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/tu/Res/btn_nosel.png);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>1311</width>
     <height>761</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="m_pLb_Verm">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>491</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;font-size: 55px;font: bold;background-color: transparent;</string>
     </property>
     <property name="text">
      <string>当前版本</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Confirm">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>440</y>
       <width>371</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tu/Res/sj.png);
background-repeat: no-repeat;
background-position: center;
background-origin: content-box; /* 确保背景图和文本区域分离 */
text-align: center;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>361</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;font-size: 55px;font: bold;background-color: transparent;</string>
     </property>
     <property name="text">
      <string>SPC20220408</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;font-size: 55px;font: bold;background-color: transparent;</string>
     </property>
     <property name="text">
      <string>OTV_01_05</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_pBtn_Cancel">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>630</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tu/Res/return.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tu/Res/sj-1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="m_pLb_Msg">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>1151</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;font-size: 55px;font: bold;background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
